### PR TITLE
feat: improved auth logs interface

### PR DESCRIPTION
**Pull request by Ziinc**
Improved selection renderers and column renderers, for auth logs. Includes workarounds for metadata parsing on client side, based on event message.

Depends on #9713 

https://user-images.githubusercontent.com/22714384/197183569-eca83fd0-de99-4815-bc89-35751a5e4754.mov

### CODE REVIEW

File: studio/components/interfaces/Settings/Logs/LogSelectionRenderers/AuthSelectionRenderer.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  error: string
+  method: string
+  level: string
+  msg: string
+  path: string
+  status: number
+}
+const AuthSelectionRenderer = ({ log }: { log: PreviewLogData }) => {
+  let parsed: AuthEventParsed | undefined
+  try {
```
Comment (joshenlim) [inline studio/components/interfaces/Settings/Logs/LogSelectionRenderers/AuthSelectionRenderer.tsx:22]: we have a tryParseJson helper - can use that instead?

File: studio/components/interfaces/Settings/Logs/LogColumnRenderers/AuthColumnRenderer.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  SeverityFormatter,
+  TextFormatter,
+  TimestampLocalFormatter,
+} from '../LogsFormatters'
+
+export default [
+  {
+    formatter: (data: { row: PreviewLogData }) => {
+      let parsed: AuthEventParsed | undefined
+      try {
```
Comment (joshenlim) [inline studio/components/interfaces/Settings/Logs/LogColumnRenderers/AuthColumnRenderer.tsx:14]: likewise we have a tryParseJson helper - can use that instead?

**Review by joshenlim — APPROVED** [review]
small comment but lgtm otherwise!

**Comment by Ziinc** [comment]
updated, thanks @joshenlim 🚀 🚢

**Pull request merged**